addons/viewport/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@storybook/addon-viewport",
+  "version": "3.2.0",
+  "description": "Storybook addon to change the viewport size to mobile",
+  "main": "dist/index.js",
+  "keywords": [
+    "storybook"
+  ],
+  "scripts": {
+    "build": "babel --source-maps --out-dir dist src",
```

Comment:
remove all but the `prepublish`

Reply:
👍 i wasn't sure of the dev process prior. i'll cleanup so it only works in kitchen sink.